config/locales/en.yml

```diff
@@ -191,6 +191,20 @@ en:
 
       <h2 class="govuk-heading-m">If you didn’t expect to see this page</h2>
       <p class="govuk-body">You may be seeing this page because you’ve turned off cookies in your browser. You’ll need to <a class="govuk-link" href="https://ico.org.uk/your-data-matters/online/cookies/" rel="external" target="_blank">turn cookies on</a> before you can use this service.</p>
+  sms:
+    confirmation:
+      body_text: |
+        Hello %{first_name} %{last_name} - we’ve received your registration.
```

Comment:
Do we want to limit the length of `first_name` or `last_name`, and also remove any special characters (e.g. `.`). Spammers could use this to send texts to people with e.g. some spam message, perhaps containing a link. This isn't specific to the SMS though..

Reply:
I'm hesitant to do this, since some names can be rather long and include special characters [1].  We have around 300 characters remaining in the SMS, so I don't think length is an issue.

Maybe we should have another story to perform some validation on these fields when the data is actually entered so anything that looks like a URL is disallowed, thereby preventing phishing attempts using this form?

[1]: https://www.bbc.co.uk/news/world-us-canada-52557291